test/crd.go

```diff
@@ -269,20 +270,15 @@ func createPipelineRunServiceAccount(namespace string) *corev1.ServiceAccount {
 
+	decoded, err := base64.StdEncoding.DecodeString(encoedDockercred)
```

Comment:
hm, this is just base64 encoded right? itll be harder for ppl to find it, but they'll still have direct access to the private key if they decoded this?

Reply:
I am referring [this](https://github.com/knative/build/blob/master/test/docker-basic/0-secret.yaml) test build.

Reply:
btw in #158 @tanner-bruce also had to make a service account available, instead he relied on the user to provide one (but added docs about it), i think that might be the best compromise

Reply:
also reached out to Prow folks and @srinivashegde86 pointed out that Prow can make secrets available via volume mounts: https://github.com/knative/test-infra/blob/master/ci/prow/config.yaml#L241 which would mean first adding them to the cluster, then updating the prow config to mount it, which seems okay - but I think the solution from #158 is better and will work for other ppl's clusters too, without giving them this private key

Reply:
This secret is currently used in build e2e test as well. For now we are okay to leave this .